core/base/inc/TPluginManager.h

```diff
@@ -155,7 +155,7 @@ friend class TPluginManager;
       // resource ... and both SetParams and Execute ends up taking the lock
       // individually anyway ...
 
-      R__LOCKGUARD2(gInterpreterMutex);
+      R__LOCKGUARD(gInterpreterMutex);
```

Comment:
Should we also apply the same fix to all occurrences of R__LOCKGUARD_IMT2 and replace them by R__LOCKGUARD_IMT?